src/main/java/nl/tudelft/ti2206/group9/gui/SettingsScene.java

```diff
@@ -31,22 +29,25 @@
 		SETTING_SOUND
 	 }
 
+	/** Toggle width. */
+	private static final int TOGGLE_WIDTH = 150;
+	/** Toggle text size. */
+	private static final int TOGGLE_TEXT = 16;
```

Comment:
Naming is slightly ambiguous. This should be TOGGLE_TEXT_SIZE or simply TOGGLE_SIZE.

< review, naming